amida-tech/recruitment-registry
# Important: Take a look in the "survey.js" file I checked in and look for

#### What's this PR do?
This PR demonstrates how we're going to be handling the various kinds of input types. 

#### Related JIRA tickets:
None.

#### How should this be manually tested?
Go to the /api folder and run "node syncAndLoadAlzheimer.js" then log in and go to the Examples survey. Look in the selectedSurvey for the first question and you'll see a group called "meta." We can read that to make decisions regarding what components to use. See the "survey.js" for how you can input whatever values you'd like. 

#### Any background context you want to provide?
His father invented the question mark...

#### Screenshots (if appropriate):
the "meta" property. This can be used to set whatever we want for data
input types, as per the method demonstrated.
